src/applications/hca/enrollment-status-helpers.jsx

```diff
@@ -619,195 +541,217 @@ export function getFAQBlock3(enrollmentStatus) {
-            review, please call our enrollment case management team at
-            877-222-VETS (
-            <a className="help-phone-number-link" href="tel:1-877-222-8387">
-              877-222-8387
-            </a>
-            ).
-          </p>
-          <p>
-            We only recommend applying again if you’ve already worked with our
-            enrollment case management team, and they’ve advised you to reapply.
-          </p>
-        </>
-      );
-      break;
+  const faqBlockReapply4 = (
+    <>
+      <h4>Can I apply again?</h4>
+      <p>
+        Yes. If you have questions about how to complete your application,
+        please {callOurTeam}.
+      </p>
+    </>
+  );
 
-    case HCA_ENROLLMENT_STATUSES.activeDutyHasApplied:
-    case HCA_ENROLLMENT_STATUSES.pendingOther:
-    case HCA_ENROLLMENT_STATUSES.pendingUnverified:
-      content = (
-        <>
-          <h4>Should I apply again?</h4>
-          <p>
-            <strong>
-              No. We’re in the process of reviewing your current application,
-              and submitting a new application won’t affect our decision.
-            </strong>{' '}
-            If you’d like to talk about your current application, please call
-            our enrollment case management team at 877-222-VETS (
-            <a className="help-phone-number-link" href="tel:1-877-222-8387">
-              877-222-8387
-            </a>
-            ).
-          </p>
-          <p>
-            We only recommend applying again if you’ve already worked with our
-            enrollment case management team, and they’ve advised you to reapply.
-          </p>
-        </>
-      );
-      break;
+  const faqBlockReapply5 = (
+    <>
+      <h4>Should I just submit a new application with all my information?</h4>
+      <p>
+        <strong>
+          No. We’re in the process of reviewing your current application, and
+          submitting a new application won’t affect our decision.
+        </strong>{' '}
+        To get help providing the information we need to complete our review,
+        please {callOurTeam}.
+      </p>
+      <p>
+        We only recommend applying again if you’ve already worked with our
+        enrollment case management team, and they’ve advised you to reapply.
+      </p>
+    </>
+  );
 
-    default:
-      break;
-  }
-  return content;
+  const faqBlockReapply6 = (
+    <>
+      <h4>Should I apply again?</h4>
+      <p>
+        <strong>
+          No. We’re in the process of reviewing your current application, and
+          submitting a new application won’t affect our decision.
+        </strong>{' '}
+        If you’d like to talk about your current application, please{' '}
+        {callOurTeam}.
+      </p>
+      <p>
+        We only recommend applying again if you’ve already worked with our
+        enrollment case management team, and they’ve advised you to reapply.
+      </p>
+    </>
+  );
+  const faqMap = {
```

Comment:
I think this is a lot easier to read. It's creating fragments for all of the options on every render but I don't think that's an issue on this page. If you run into perf issues you could move this out and turn these into functions e.g. 
```
[HCA_ENROLLMENT_STATUSES.activeDutyHasApplied]: () => wrapJSXInKeyedFragment([
	      faqBlock8,
	      faqBlock10,
	      faqBlockReapply6,
	    ]),
...
return faqMap[enrollmentStatus]();
```

Reply:
I guess that's a bad suggestion because the fragments are created when they're assigned.